lib/formfields/admin/domains/formfield.domains_add.php

```diff
@@ -234,6 +234,14 @@
 						'value' => '1',
 						'checked' => false
 					],
+					'http3' => [
+						'visible' => !empty($ssl_ipsandports) && Settings::Get('system.webserver') != 'lighttpd' && Settings::Get('system.http3_support') == '1',
```

Comment:
this should be changed from
```diff
...
- Settings::Get('system.webserver') != 'lighttpd'
+ Settings::Get('system.webserver') == 'nginx'
...
```

as i doubt lighttp does this....

Reply:
Ah yeah, just copied this from http2 and Apache doesn't support h3. 🤦🏼